cmd/juju/charmhub/find_test.go

```diff
@@ -35,11 +36,45 @@ func (s *findSuite) TestRun(c *gc.C) {
 	defer s.setUpMocks(c).Finish()
 	s.expectFind()
 	command := &findCommand{api: s.api, query: "test"}
+	cmdtesting.InitCommand(command, []string{})
 	ctx := commandContextForTest(c)
 	err := command.Run(ctx)
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *findSuite) TestRunJSON(c *gc.C) {
+	defer s.setUpMocks(c).Finish()
+	s.expectFind()
+	command := &findCommand{api: s.api, query: "test"}
+	cmdtesting.InitCommand(command, []string{"--format", "json"})
+	ctx := commandContextForTest(c)
+	err := command.Run(ctx)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(cmdtesting.Stdout(ctx), gc.Equals, `[{"type":"object","id":"charmCHARMcharmCHARMcharmCHARM01","name":"wordpress","publisher":"Wordress Charmers","summary":"WordPress is a full featured web blogging tool, this charm deploys it.","version":"1.0.3","series":["bionic"],"store-url":"https://someurl.com/wordpress"}]
+`)
+}
+
+func (s *findSuite) TestRunYAML(c *gc.C) {
+	defer s.setUpMocks(c).Finish()
+	s.expectFind()
+	command := &findCommand{api: s.api, query: "test"}
+	cmdtesting.InitCommand(command, []string{"--format", "yaml"})
+	ctx := commandContextForTest(c)
+	err := command.Run(ctx)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(cmdtesting.Stdout(ctx), gc.Equals, `
+- type: object
+  id: charmCHARMcharmCHARMcharmCHARM01
+  name: wordpress
+  publisher: Wordress Charmers
+  summary: WordPress is a full featured web blogging tool, this charm deploys it.
+  version: 1.0.3
+  series:
+  - bionic
+  storeurl: https://someurl.com/wordpress
```

Comment:
We should be consistent.  The json and yaml output are different labels right now.  Most structs have it as "store-url"